src/foraging_gui/Foraging.py

```diff
@@ -1771,7 +1771,10 @@ def _Open(self):
                 self.ShowBasic.setText(Obj['Other_BasicText'])
             # Set newscale position to last position
             if 'B_NewscalePositions' in Obj:
-                last_positions=Obj['B_NewscalePositions'][-1]
+                try:
```

Comment:
If the problem is that B_NewscalePositions is empty, it would be better to check for it being empty in a if/else block, rather than using a try/except block.

Reply:
I agree that it's better to use if/else. You can get a full understanding about what's happening using if/else instead of  try/except.